pkg/controller/controller_binding_test.go

```diff
@@ -76,10 +76,6 @@ func TestReconcileServiceBindingNonExistingServiceInstance(t *testing.T) {
 	assertNumberOfActions(t, actions, 1)
 
 	// There should only be one action that says it failed because no such instance exists.
-	updateAction := actions[0].(clientgotesting.UpdateAction)
-	if e, a := "update", updateAction.GetVerb(); e != a {
-		t.Fatalf("Unexpected verb on actions[0]; %s", expectedGot(e, a))
-	}
```

Comment:
Note that the assertUpdateStatus() in the next line already checks the verb, that's why I deleted these lines